CONTRIBUTING.md

```diff
@@ -1,34 +1,48 @@
+## 🎨 Design contributions
 
-For folks who'd like to be involved in our regular development conversations, please feel free to join us in the WordPress make chat (https://make.wordpress.org/chat/). You'll find us in the #openverse channel, of course! We have weekly community meetings on Tuesdays at 15:00 UTC (https://everytimezone.com/s/d1d42c7b). All are welcome! During the meetings we discuss ongoing work and assign new issues. Attendance is not mandatory to contribute, however. Any issues labeled as "good first issue" or "help wanted" in our repositories are all up for grabs. Just add a comment with `@WordPress/openverse-maintainers` on the issue with questions or requesting the issue be assigned to you when you're ready to work on it. We also regularly discuss Openverse development issues outside the meeting times (like debugging problems we're having while working through an issue).
+### How to contribute with design
 
-You can also keep in touch with [progress](https://github.com/orgs/WordPress/projects/3) and the latest updates to the project with our [WordPress contributor group](https://make.wordpress.org/openverse/).
+If you'd like to contribute to the design or front-end, feel free to propose a solution to an existing problem labeled with [Needs Design](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+design%22), or share an idea if you think it meets Openverse's goals.
+
+The [WordPress Design team](http://make.wordpress.org/design/) uses [Figma](https://www.figma.com/) to collaborate and share work for all WordPress projects. If you are not familiar with designing for WordPress, please carefully read the [design handbook](https://make.wordpress.org/design/handbook/). Once you have a [WordPress Slack](https://make.wordpress.org/chat/) account, join the [#design channel](http://wordpress.slack.com/messages/design/) and ask the team to set you up with a free Figma account.
+
+This will give you access to [all projects and files](https://www.figma.com/files/team/642003996626140596/WordPress.org?fuid=968867265893371002) used in WordPress.
+
+Before designing a proposal, browse the [Design Library](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=0%3A1) file to understand how Openverse has been built, and take a look at the [created work](https://www.figma.com/files/project/31962071/Openverse?fuid=968867265893371002) to get a glance at how design ideas are made. The design onboarding section in the design library file might miss documentation as it is always under construction, yet ask on [#design channel](http://wordpress.slack.com/messages/design/) for clarity.
 
-## 📖 Translations
+Once you are done and ready to share your idea, create a design issue and fill the content structure template. Please be as specific and concise as possible and feel free to add mockups, prototypes, videos, sketches, and anything that makes your idea easy to understand.
```

Comment:
```suggestion
Once you are done and ready to share your idea, [create an issue with the `design` label and fill in the template](https://github.com/WordPress/openverse-frontend/issues/new?assignees=&labels=🖼️+aspect%3A+design%2C✨+goal%3A+improvement%2C+🚦+status%3A+awaiting+triage&template=feature_request.md&title=<Replace+this+with+actual+title>). Please be as specific and concise as possible and feel free to add mockups, prototypes, videos, sketches, and anything that makes your idea easier to understand.
```

This link should help direct folks to the correct place to create the issue (and it pre-fills the design label :partying_face:)

Reply:
Oh yes! I forgot that I wanted to propose the creation of a `design` template for issues. I should open that discussion now.